### ISSUE TITLE
appveyor: capabilities.c: syntax error: '}'

### ISSUE THREAD

**Issue by kroeckx** [openssl/openssl#13536]
The minimal target on appveyor is failing with:
```
..\providers\common\capabilities.c(162): error C2059: syntax error: '}'
..\providers\common\capabilities.c(174): warning C4034: sizeof returns 0
```

For a full log see: https://ci.appveyor.com/project/openssl/openssl/builds/36510326/job/7tgcvtncedaj796i

**Comment by levitte** [comment]
I suspect the compiler doesn't like trailing commas...

**Comment by paulidale** [comment]
Or everything has been omitted and the array is empty.

**Comment by levitte** [comment]
Ahhh, minimal build, that makes sense

**Comment by mattcaswell** [comment]
Fix in #13549
